flink-kubernetes/src/main/java/org/apache/flink/kubernetes/utils/Constants.java

```diff
@@ -25,6 +25,9 @@
     public static final String API_VERSION = "v1";
     public static final String APPS_API_VERSION = "apps/v1";
 
+    public static final String DNS_PLOICY_DEFAULT = "ClusterFirst";
```

Comment:
I am not sure the default value of `dnsPolicy` is `Default` or `ClusterFirst`.

Reply:
dafult value is `ClusterFirst`

https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-s-dns-policy